base_external_dbsource_firebird/README.rst

```diff
@@ -0,0 +1,77 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+===================================
+External Database Source - Firebird
+===================================
+
+This module extends ``base_external_dbsource``, allowing you to connect to
+foreign Firebird databases.
+
+Installation
+============
+
+* Install ``fdb`` python library
+
+Configuration
+=============
+
+Database sources can be configured in Settings > Configuration -> Data sources.
+
+
+Usage
+=====
+
+To use this module:
+
+* Go to Settings > Database Structure > Database Sources
+* Click on Create to enter the following information:
+
+* Datasource name 
+* Pasword
+* Connector: Choose the database to which you want to connect
+* Connection string : Specify how to connect to database
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/149/10.0 for server-tools
+
+Known issues / Roadmap
+======================
+
+* Setting ``metadata`` to ``True`` in ``execute_fdb`` will do nothing.
+* ``execute`` is susceptible to SQL injection.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/server-tools/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Daniel Reis <dreis.pt@hotmail.com>
+* Maxime Chambreuil <maxime.chambreuil@savoirfairelinux.com>
+* Gervais Naoussi <gervaisnaoussi@gmail.com>
+* Dave Lasley <dave@laslabs.com>
```

Comment:
@lasley Please, add me as Contributors. The firebird support was provide by me.
https://github.com/OCA/server-tools/pull/623